tests/phpunit/elementor/includes/test-tracker.php

```diff
@@ -40,7 +42,7 @@ public function test_get_settings_advanced_usage() {
 		// Load font_awesome_support settings.
 		Plugin::$instance->icons_manager->register_admin_settings( Plugin::$instance->settings );
 
-		update_option( 'elementor_css_print_method', 'internal' );
+		Site_CSS_Print_Method::set( 'internal' );
```

Comment:
Maybe that could be useful? Like `set_on()` / `set_off()`

```suggestion
		Site_CSS_Print_Method::set_internal();
```